docs/docs/testing-components-with-graphql.md

```diff
@@ -254,16 +254,16 @@ the Header component as:
 
 ```jsx:title=src/components/header.js
 import React from "react"
-import { StaticQuery, graphql } from "gatsby"
+import { useStaticQuery, graphql } from "gatsby"
 
 export const PureHeader = ({ data }) => (
   <header>
     <h1>{data.site.siteMetadata.title}</h1>
   </header>
 )
 
-export const Header = props => (
-  const data = useStaticQuery(graphql`
+export const Header = props => {
+  const { data } = useStaticQuery(graphql`
```

Comment:
```suggestion
  const data = useStaticQuery(graphql`
```
It doesn't need to be destructured

Reply:
thanks, Vladar i made the change that you have suggested